src/test/shell/bazel/android/aidl_integration_test.sh

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x
```

Comment:
Please take these debug commands out if they're only for debug.

Reply:
Done.